.codecov.yml

```diff
@@ -0,0 +1,25 @@
+codecov:
+  notify:
+    require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "20...100"
+  status:
+    project: no
+    patch: false
+    changes: false
```

Comment:
Just out of curiosity. Is there some docs for these attributes?

Reply:
Well, the Codecov documentation should have it.

Reply:
https://docs.codecov.com/docs/codecov-yaml

Reply:
Are `project` and `patch` attributes set correctly? I was looking at https://docs.codecov.com/docs/commit-status#disabling-a-status

Reply:
AFAIK, both should works but no problem to change it to `off`.

Reply:
```sh
➜ curl --data-binary @.codecov.yml https://codecov.io/validate

Valid!

{
  "codecov": {
    "notify": {},
    "require_ci_to_pass": false
  },
  "coverage": {
    "precision": 2,
    "round": "down",
    "range": [
      20.0,
      100.0
    ],
    "status": {
      "project": false,
      "patch": false,
      "changes": false
    }
  },
  "parsers": {
    "gcov": {
      "branch_detection": {
        "conditional": true,
        "loop": true,
        "method": false,
        "macro": false
      }
    }
  },
  "comment": {
    "layout": "reach,diff,flags,tree",
    "behavior": "default",
    "require_changes": false
  }
}
```

Never mind I'll not change it... I checked and the validator seems to be fine with it.

Reply:
OK thanks and no problem, but now the range `range: "70...75"` seems to strict to me. :) I would go something like `40..75` or `30..75`

Reply:
OK. Let me change it to 30...75. :)